ansible/live_mode.c

```diff
@@ -18,8 +18,25 @@ void set_slew_icon(bool display) {}
 void set_metro_icon(bool display) {}
 void init_live_mode(void) {}
 void set_live_mode(void) {}
+
+uint8_t get_live_sub_mode() {
+    return 0;
+}
+
 void set_live_submode(u8 submode) {}
+
+void select_dash_screen(uint8_t screen) {}
+
+void print_dashboard_value(uint8_t index, int16_t value) {}
```

Comment:
while the dashboard itself is not used in live mode, it's possible somebody will use `PRINT` ops simply to store values, so this needs to store values and `get_dashboard_value` needs to return them